src/main/java/com/example/todobackend/controllers/TodoController.java

```diff
@@ -23,22 +24,22 @@ public TodoController(TodoRepository todoRepository){
         this.todoRepository = todoRepository;
     }
 
-    @GetMapping("/todo")
+    @GetMapping
     public List<Todo> getTodos() {
         return this.todoRepository.findAll();
     }
 
-    @GetMapping("/todo/{id}")
+    @GetMapping("/{id}")
     public Todo getTodo(@PathVariable("id") String id) {
         return this.todoRepository.findById(id).get();
     }
 
-    @PostMapping("/todo")
+    @PostMapping
     public Todo saveTodo(@RequestBody Todo todo){
         return this.todoRepository.save(todo);
     }
 
-    @DeleteMapping("/todo/delete/{id}")
+    @DeleteMapping("/{id}")
```

Comment:
Need not add extra `delete` since `DeleteMapping` already lets us know about that